swim/member.go

```diff
@@ -50,11 +50,11 @@ type Member struct {
 }
 
 // suspect interface
-func (m Member) address() string {
+func (m *Member) address() string {
```

Comment:
Interesting. If we remove the lock, we could satisfy the linter while touching less code elsewhere.